### ISSUE TITLE
Jasper report version

### ISSUE THREAD

**Issue by matteoturra** [gmazelier/gradle-jasperreports#12]
Is there a parameter to specify the jasper version?

**Comment by tkofford** [comment]
Yeah, I just got tripped up by this. Using latest version of jaspersoft studio 6.12.0 to modify/enhance existing jrxml report templates. However, when I integrated those templates in my main spring boot project, this plugin fails to compile the modified reports, throwing the following error: 

`Attribute 'textAdjust' is not allowed to appear in element 'textField'`

Apparently, jaspersoft changed/renamed a textField property. What used to be `isStretchWithOverflow="true"` is now `textAdjust="StretchHeight"` as of jasper reports version 6.11.0. 

However, this plugin is hard-coded to use jasper reports version 6.10.0. This is really a jaspersoft issue, but the inflexibility to specify a jasper reports version for the plugin makes it a real issue.

**Perhaps there's a workaround?** Otherwise, I would like to see a parameter to specify the jasper reports version in this plugin.

**Comment by viktorKhan** [comment]
I'm experiencing the same issue, is there any plan to solve this issue?

**Comment by klaskoski** [comment]
I had the same issue. Got it working by forcing the newer jasper version.
It is not great but works for me
`buildscript {
    configurations.all {
        resolutionStrategy {
            force "net.sf.jasperreports:jasperreports:6.11.0"
        }
    }
}`
